lnd_test.go

```diff
@@ -13852,6 +13852,124 @@ func testChannelBackupRestore(net *lntest.NetworkHarness, t *harnessTest) {
 	}
 }
 
+// testTLSAutoRegeneration creates an expired TLS certificate, testing that a new TLS certificate pair is regenerated when the old pair expires
+func testTLSAutoRegeneration(lnNet *lntest.NetworkHarness, t *harnessTest) {
+	certPath := lnNet.Alice.Cfg.TLSCertPath
```

Comment:
instead of making `cfg` public by naming it `Cfg`, perhaps we can instead define getters for `TLSCertPath` and `TLSKeyPath`? Then we would no longer require the prior commit